shipment.go

```diff
@@ -110,6 +112,8 @@ type ShipmentOptions struct {
 	PrintCustom1Code         string     `json:"print_custom_1_code,omitempty"`
 	PrintCustom2Code         string     `json:"print_custom_2_code,omitempty"`
 	PrintCustom3Code         string     `json:"print_custom_3_code,omitempty"`
+	RegisteredMail           bool       `json:"registered_mail,omitempty"`
+	RegisteredMailAmount     *float64   `json:"registered_mail_amount,omitempty"`
```

Comment:
Does this need to be a pointer? `registered_mail = true` and we don't send `registered_mail_amount`, is `0` not the default?